build/marketplace/MarketPlace.sol

```diff
@@ -34,6 +34,8 @@ contract MarketPlace {
     address public admin;
     /// @notice address of the deployed redeemer contract
     address public immutable redeemer;
+    /// @notice flag that determines if pools may be used
+    bool public paused;
```

Comment:
this, obvs, is all markets

do we want this blanket pause or an individual way to pause a specific market?

Reply:
I'm not sure. I'll ask product.